dash-renderer/src/actions/index.js

```diff
@@ -128,6 +131,8 @@ export function redo() {
             })
         );
 
+        dispatch(setAppIsReady());
```

Comment:
Do undo and redo really need this? They're both returning to a state you've already been in at some point, so doesn't that mean you've already waited for that state to be ready?

Reply:
No. Being too defensive here.